Core/GDCore/Project/Layer.h

```diff
@@ -177,6 +221,11 @@ class GD_CORE_API Layer {
  private:
   gd::String name;                  ///< The name of the layer
   bool isVisible;                   ///< True if the layer is visible
+  bool isLightingLayer;             ///< True if the layer has lighting properties
+  bool syncWithBaseLayer;           ///< True if the layer automatically follows the base layer

```

Comment:
I know I'm a pain to ask for renaming, but could we rather name this (and associated methods): `followBaseLayerCamera`?

"Follow" indicates the notion of spatial position, while "sync" can be about anything.

Reply:
@4ian should I rename the getter/setter as GetFollowBaseLayerCamera/SetFollowBaseLayerCamera?

Reply:
Yes please! :)

Reply:
@4ian I'll be renaming the functions as:

- SetAsLightingLayer()

- IsLightingLayer()

- SetToFollowBaseLayerCamera()

- IsFollowingBaseLayerCamera()

Are they fine? Or suggest something better :)

Reply:
For the setters, I would go for:
* SetLightingLayer (or SetIsLightingLayer), with a bool as argument
* SetFollowBaseLayerCamera (same, with a bool as argument)

the getter sounds fine